config/crd/bases/kamaji.clastix.io_tenantcontrolplanes.yaml

```diff
@@ -112,6 +112,11 @@ spec:
                           port: 8132
                           version: v0.0.32
                         properties:
+                          address:
+                            description: The address which Konnectivity server is
+                              listening will listen on.  If not set this value defaults
+                              to the address of the TenantControlPlane.
+                            type: string
```

Comment:
Although specifying an address, this PR doesn't provision the `Service` required to expose the Konnectivity server.

Are you provisioning manually? If this is the case, I'd avoid extending the CRD with this field and work rather at the extra args level, by allowing the address override.

Reply:
As said before, let's drop this.